### ISSUE TITLE
Battle Pass Bonus Chest Problem

### ISSUE THREAD

**Issue by TheGameLord2** [ValveSoftware/Dota2-Gameplay#4681]
### Description

When I was kicking my bonus chest, It rose up to 2000 but did not give me another chance to kick again and gave only 1000 point.
In the previous week I have also gotten only 1000 points from bous chests but I thought It was because I did get them in the later weeks but know I am unsure. I want help ASAP.

**Comment by stirwick** [comment]
I have experienced the exact same thing. This happened to me in all previous weeks as well as this one.

**Comment by mujigaegeomi** [comment]
I also experienced this problem, but I reached 4000 points and I was dropped to 1000

**Comment by S1L3NZE** [comment]
I also experienced this problem, but I reached 2000 points and I was dropped to 1000))

**Comment by joaorzc** [comment]
I also experienced this problem. Kicked it until 2000. It showed some sort of loading interface, then got back to 1000.
Here's a video showing it in real time.

https://user-images.githubusercontent.com/96877921/199877897-7991ce79-fcc8-4e64-8529-06cd02e59eed.mp4

My ID is https://steamcommunity.com/id/mudigov/

**Comment by siavashdarmani** [comment]
yeah i have the same problem and it seems you dont get all the battle points for me i only get the 1000 very frustrating i stopped playing any quest or cavern and if i had the option i would have got the refund for my whole battle pass.

**Comment by deadpool5157** [comment]
> ### Description
> When I was kicking my bonus chest, It rose up to 2000 but did not give me another chance to kick again and gave only 1000 point. In the previous week I have also gotten only 1000 points from bous chests but I thought It was because I did get them in the later weeks but know I am unsure. I want help ASAP.

iam have the same problem i want my levels back

**Comment by Enternalwitness101** [comment]
Also having the same issue. We need this resolved. or to be compensated back for this, as this is a part of a bpass purchased by the community. Thanks.

**Comment by Dean-Wolf** [comment]
same here opened 4 chests yesterday in the evening and they went up to 2 and 4 k after 2 kicks but then i got freezing screen and it saves only 1k for each chest its so bugged plz fix this and give us chests back...

**Comment by TheGameLord2** [comment]
Has anyone gotten an update?

**Comment by Dreco17** [comment]
two weeks already and no fix from valve, shitty BP
